ui/DesignSystem/Component/Rad.svelte

```diff
@@ -41,18 +46,31 @@
 </style>
 
 <div class="wrapper" {style} on:mouseenter={enter} on:mouseleave={leave}>
-  <div class="amount {variant}">
-    {#if variant === 'deposit'}
-      <Icon.LockSmall
-        style="fill: var(--color-foreground-level-5); margin-right: 2px;" />
+  {#if size === 'regular'}
```

Comment:
Lots of nested `if` `else`'s; could use a little DRY magic here

Reply:
Yeah good point. We'll be reworking this when we remove deposits.